### ISSUE TITLE
Feature request - Duplicate operator with connections

### ISSUE THREAD

**Issue by MarcoMazzo** [still-scene/t3#49]
Duplicate the selected operator maintaining the incoming connections:
- a method could be "select -> Alt+drag " the operator/s;
- or maybe a combination of key like "ctrl + shift + D".

(i'm referencing from other software behavior)

**Comment by pixtur** [comment]
I completely agree that we need this function. Added to todo-list for next release.

**Comment by ScreenDream** [comment]
Yeah, Houdini does it with Alt drag and I find that very handy and immediate feeling, since you can directly place your operator in the same move.

**Comment by pixtur** [comment]
Oh. That's a good idea!
